badparking/core/models.py

```diff
@@ -11,3 +13,23 @@ def __unicode__(self):
     class Meta:
         verbose_name = "Тип порушення"
         verbose_name_plural = "Типи порушень"
+
+
+class Claim(models.Model):
+    license_plates = models.CharField("Номери автомобілів", max_length=50, blank=True)
+    longitude = models.CharField("Довгота", max_length=50, blank=True)
+    latitude = models.CharField("Широта", max_length=50, blank=True)
+    city = models.CharField("Місто", max_length=30, blank=True)
+    address = models.CharField("Адреса", max_length=150, blank=True)
+    created_at = models.DateTimeField(editable=False, auto_now_add=True)
+
+    user = models.ForeignKey(User, verbose_name="Користувач")
+    crimetypes = models.ManyToManyField(CrimeType, verbose_name="Порушення")
+    images = models.ManyToManyField(MediaFileModel, blank=True, verbose_name="Зображення")
+
+    def __unicode__(self):
```

Comment:
Primary key doesn't tell anything to a human. It just doesn't make sense to expose a custom `__str__` then. You could include a crime type, license plate, date and user for instance — the info you'd want to a have visible immediately when you serialize to a human-readable string.